modules/core/src/main/scala/org/tessellation/l0/infrastructure/snapshot/SnapshotService.scala

```diff
@@ -2,46 +2,82 @@ package org.tessellation.l0.infrastructure.snapshot
 
 import cats.data.NonEmptySet
 import cats.effect.kernel.Async
```

Comment:
if you see an import from `kernel` please remove that, to have:
```
import cats.effect.Async
```

Reply:
Auto-import from IntelliJ. I will try to ban auto-imports from kernel in IDE